src/styles/sass_style/header.scss

```
@@ -0,0 +1,57 @@
+@import "sass_style/variable.scss";
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-weight: 500;
+  font-size: 12px;
+  color: #000;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  padding: 23px 0;
+  text-decoration: none;
+  color: #060b35;
+  white-space: nowrap;
+  text-transform: uppercase;
+  &:hover {
+    color: $blueColor;
+  }
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.logo {
+  display: flex;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+}
+
+.is-active::after {
```

Better use pseude-element nesting

```suggestion
.is-active {
  &::after {}
}
```